### PR TITLE
Try to write SVGs in the page

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

File: src/writer.jl [hunk 1]
```diff
@@ -438,7 +438,7 @@ function render_mime(io::IO, mime::MIME"image/svg+xml", node, element, page, doc
     if !isnothing(location)
         image_text = replace(image_text, image_text[location] => "")
     end
-    println(io, image_text)
+    println(io, "<img src=\"data:image/svg+xml;\"" * base64encode(image_text) * "\"/>")
```
[the remark this answers is not in the document]
Comment (asinghvi17) [reply src/writer.jl:441]: ah we just need to import it

**Comment by lazarusA** [comment]
Screenshot 2024-03-01 at 15 05 47

what? the output looks good, but no image :D

File: src/writer.jl (hunk 1 above)
[the remark this answers is not in the document]
Comment (asinghvi17) [reply src/writer.jl:441]:
```suggestion
    println(io, "<img src=\"data:image/svg+xml;base64,\"" * base64encode(image_text) * "\"/>")

**Comment by lazarusA** [comment]
nice. It works nicely. Thanks. Should we merge then?

**Pull request merged**